src/libs/ReportActionsUtils.ts

```diff
@@ -1728,13 +1729,24 @@ function isCardIssuedAction(reportAction: OnyxEntry<ReportAction>) {
     return isActionOfType(reportAction, CONST.REPORT.ACTIONS.TYPE.CARD_ISSUED, CONST.REPORT.ACTIONS.TYPE.CARD_ISSUED_VIRTUAL, CONST.REPORT.ACTIONS.TYPE.CARD_MISSING_ADDRESS);
 }
 
-function getCardIssuedMessage(reportAction: OnyxEntry<ReportAction>, shouldRenderHTML = false) {
-    const assigneeAccountID = (getOriginalMessage(reportAction) as IssueNewCardOriginalMessage)?.assigneeAccountID;
+function getCardIssuedMessage(reportAction: OnyxEntry<ReportAction>, shouldRenderHTML = false, policyID = '-1') {
```

Comment:
Why -1 and not 0 or null? (same in all the ones below)

Reply:
Because the ```policyID``` prop of the ```report``` is ```policyID?: string```, so I need to add the fallback value as '-1' to fix the type error based on the docs: https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#default-value-for-inexistent-ids

Reply:
Posted about this here https://expensify.slack.com/archives/C01GTK53T8Q/p1728329641957179 and we will be undoing this, but I'll go ahead and merge this